app/javascript/packages/document-capture/components/document-capture-review-issues.tsx

```diff
@@ -10,7 +10,6 @@ import { useI18n } from '@18f/identity-react-i18n';
 import UnknownError from './unknown-error';
 import TipList from './tip-list';
 import DocumentSideAcuantCapture from './document-side-acuant-capture';
-import DocumentCaptureAbandon from './document-capture-abandon';
```

Comment:
@aduth, based on @daviddsilvanava we may put them back in the future, intact or with some variations.

Reply:
Generally my preference is to remove all unused code even if the intent is to bring them back, since the future is uncertain and priorities can change. But I understand there's a time constraint here, so not going to insist on it.